tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/exit-to-wordpress.cy.js

```diff
@@ -0,0 +1,52 @@
+// <reference types="Cypress" />
+
+describe('Exit to WordPress', function () {
+
+    before(() => {
+        cy.setCustomerData();
+        cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/ecommerce/step/products');
+		// cy.injectAxe();
+    });
+
+    // it('Is Accessible', () => {
+	// 	cy.checkA11y();
+	// });
+
+    it("Go to the Page where Exit to WordPress Button is visible", () => {
+        var drawer_status = cy.get('.nfd-onboarding-drawer__panel-site-title-container').scrollIntoView().should('not.be.visible');
+        var noExitToWPLabel = cy.get('.nfd-onboarding-drawer__panel-back').should('not.contain', 'Exit');
+        if(drawer_status || noExitToWPLabel) {
+            cy.get('.navigation-buttons_next').click();
+        }
+        cy.url().should('not.contain', '/ecommerce/step/products');
+        cy.get('.nfd-onboarding-etw__trigger').click();
+        cy.get('.components-modal__content').should('be.visible');
+    });
+
+    it('Check if heading and paragh content exists', () => {
+        cy.get('h1.components-modal__header-heading').should('be.visible');
+        cy.get('.components-modal__content > p').should('be.visible');
+    });
+
+    it("Stay on Onboarding Page when 'X'/Continue is clicked", () => {
+        var currUrl = cy.url().should('not.contain', '/ecommerce/step/products');
```

Comment:
```suggestion
               cy.url().then((currUrl) => {
            cy.get('.components-modal__header > .components-button').click();

            cy.get('.nfd-onboarding-etw__trigger').click();
            cy.get('.nfd-onboarding-etw__buttons > .is-secondary').click();
            cy.url().should('equal', currUrl);
        });
        });
```

Reply:
This is what I meant in reference to the previous comment (can't find it looks like it was marked as resolved). Please fix the bad indentation as you push the change, thanks.